doc/application/application.rst

```diff
@@ -98,8 +109,26 @@ Follow these steps to create a new application directory. (Refer to
       cd app
       mkdir src
 
-#. Create a :file:`CMakeLists.txt` file in your application directory with the
-   following contents:
+#. Create an empty :file:`CMakeLists.txt` file in your application directory.
+
+#. Optionally include additional :file:`.conf` files in your
+   :file:`CMakeLists.txt`. By default the build system will pick up
+   :file:`prj.conf` automatically:
+
+   .. code-block:: cmake
+
+   set(CONF_FILE "prj.conf ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf")
+
+#. Optionally include a reference to your :file:`dts.overlay` in your
```

Comment:
If you add the new dts overlay section as described above, this can then be structured the same way as the above:

```
#. Optionally configure any device tree overlays needed by your application. This is
   usually not necessary; see :ref:`reference_to_details_section` below for more details.
```